src/content/pages/volunteers.mdx

```diff
@@ -0,0 +1,120 @@
+* Keeping in touch with the AV team to ensure everything is in order
+* Keeping the Code of Conduct (CoC) in mind and contact CoC committee members if needed
+
+We want to call out that room managers might not be able to attend or focus on talks for the duration of the task.
+
+## Session Chairing
+
+A session chair plays a key role in introducing talks and ensuring a seamless transition between talks scheduled during a slot. The activities that a session chair is expected to carry out include:
+
+* Helping speakers in preparing their computers for the talks
+* Introducing the speakers to the audience
+* Signalling time left to the speakers
+* Making sure talks start and finish on time
+* Moderating Q&A session and help with passing around microphones and introduce remote attendees.
+
+## Set Up/Tear Down
+
+* Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare etc.)
+* Also on the last day (Friday afternoon for the main conference, Sunday for the sprints) to clear up the venue afterward
+* This job involves walking and carrying items, some of which maybe heavy.
+
+Runners and Ad-hoc Activities
+
+There are always some tasks which we simply cannot plan beforehand. Based on what's needed in the moment, you can be expected to do one or more of the following (not an exhaustive list):
+
+* Checking rooms to ensure all the necessary resources are in place
+* Helping out with Registration Desk duties, T-Shirt Handout, or Room Manager duties
+* Moving chairs, tables, or poster walls
+* Arranging transfers between venue and airport, etc. (Rarely)
+
+# How to sign up
+
+We are currently still working out the latest details for the volunteering program. (We are trying to get more perks enabled for your volunteering time).
+Until then, you can sign up for volunteering by sending an email to [volunteers@europython.eu](mailto:volunteers@europython.eu) .
+
+<div class="text-center">
+<ButtonLink href="mailto:volunteers@europython.eu">volunteers @ europython.eu</ButtonLink>
+</div>
+
+# More information
+## Why be a EuroPython Volunteer?
+
+* To help the EuroPython organizers keep the conference affordable for everyone
+* Show your EuroPython support to the Python community (by wearing your exclusive T-Shirt)
+* Get a huge applause during the closing session of the conference
+* Get your name on our EuroPython 2024 Team page to stay there forever.
+
+## How much work is involved?
+
+* There are different tasks with different time required
+* Each task is related with the concept of slot (Usually bracketed by the breaks)
+* You can sign up to any number of them.
+* We would really appreciate if you can sign up for more than one slot.
+
+## What kind of qualifications do I need?
+
+* You should be able to speak some English.
+* If you speak additional languages, even better. Please let us know.
+* Have time between two talks? Then why not spend it as an EuroPython Volunteer on the spot?
```

Comment:
```suggestion
* Have time between two talks? Then why not spend it as a EuroPython Volunteer on the spot?
```

Reply:
Why "a" instead of "an"? For me the sounding is "Uro" which is a vowel and therefore falls into the "an" category. Which rule should I brush up on?

Reply:
So, the phonetics of "euro" is  /ˈjʊə.rəʊ/. It starts with a /j/ (a bit like "y" in "yellow") which is a consonant sound, thus it should be "a" in this case. If you say "EPS volunteer", it sounds with E ( /iː/), which is a vowel sound, where "an" should be used.

Reply:
Facepalm. Yes. Thank you.